daprdocs/content/en/developing-applications/building-blocks/state-management/query-state-store-api.md

```diff
@@ -0,0 +1,265 @@
+---
+type: docs
+title: "State Query API"
+linkTitle: "State Query API"
+weight: 600
+description: "API for querying state stores"
+---
+
+## Introduction
+
+The query API provides unified way of querying state stores regardless of underlying databases.
+
+{{% alert title="In preview" color="warning" %}}
+The state store query API is currently in [preview]({{< ref preview-features.md >}}). To enable, opt-in to `State.Query` feature.
+{{% /alert %}}
+{{% alert title="Note" color="primary" %}}
+The state store query API is implemented per component. Currently we support `MongoDB` and `Azure CosmosDB`. Support for more components is underway.
+{{% /alert %}}
+
+## Syntax of the query request
+
+Users submit query requests via HTTP POST or gRPC.
+The body of the request is the JSON map with 3 entries: `filter`, `sort`, and `pagination`
+The `filter` section specifies the query conditions in the form of a tree of key/values operations, where the key is the operator and the value is the operands.
+
+Currently we support four operations:
```

Comment:
"we" should not be used and avoid the world "currently'. See documented tone guidelines https://docs.dapr.io/contributing/contributing-docs/#style-and-tone. This should be

"The following operations are supported:"